src/main/resources/staff/templates/document/countyCourtJudgment.html

```diff
@@ -1,9 +1,253 @@
puuXd6454Vc7yit+16XYuNa1lKf8fi5J4zxbFtF3G8xc2lBud9qptLZ4jiFgDQaLD37wg4saRLotyk+57vcbBCfpkOqUqz7gVR3VeSv1xpdtCpIpj3c1ovR0nHe5biqj0xAp9XOtLZuLcpy8KL57cYMb3GCR6TfIAynyi9JTmx/+8IfP74VbRIMSbfd7C+zaY98IJhxRgQ996EOL2k/dxEb5V+uOfg/p5fvRj360f7lHcH4kV8thZefWHxeyqm/ZvC2+9Vu/dUHpyNwCAT7mMY/p+kD0s846q+/r8JoFb/kUkliqtM+1HIR3fZR/n/vc54Qw/YNdQ4i1g29Rs+niUpe6VMdjbg7St/2wz/5SN6xvzc6LMqLtGoZ4NlPdLROIo5Zr6ppH360UtpkK7UacdJSPWp7FGwmIVCEIYichglyve93r2h1/ydFbCgKhAmYAfH0JJnpNAknBX/zFXywgkrzKMLOvjodDOGaY8sbR74KUX/VVX9XIigjKmrdnK3WoBfOiFuSLUhx2nUoQ0M/eIWjpnUWuLYg6+eVauqiujxE/73K1aS31RWTWMtqKRx9nNf2gjPve976LYu96lpVpTIjk53BOV4enjlKwLnyf/eW7kzpqkzWfgQkkbKMqb4pAQgQf/vCH+4MasbZSyEaV2O3wdEbsn4yukJiLSx8by5j2GIktWrVZOgjklz5YVVeLX3s1EB2kz6xkwS5/5VzmMpeZR3Szi0V6HV3d4WYe8db6lfubDrPoVo9at/QzFiqbq+w+RHTyfUnpTkoK1ywh41Gzx1oAYbQviOOe8CGibO/raOcuD9ulHfQzt7zlLecNX+v1zVrl7tX71C37k0pK2UXn/Ub12HCRXhn0As21pqreU85P0UGEYjm62s4fKaRtd6f2yNvrDSzuLFxrIJhKKjXVrr8W+Rah9OJOHOb7FtOOaHDylQUtd5nF+3dccbLQtm8aWBBLZ+ELiBz9iCKLSDqMmJcr1iww0+/qRsPOwyCBAYj2vT5yp/WuCL7PBJTvXe5yl+m2t71ti97VlVBFm+Vfs0a3rxC/66uOFvd+RLpEv9rGvAV4b3EL5GPPvP6pdUwLIbxPnd3vN0jdymq6t2EQqBC26Bd9nPA1612R1oVQmgVaZdJshgQZlddNvA8C1TNtMBq+4hWv6HZoS338RTlMmGtpJiDaFLbZX23SWTz96U/vWYiJ/7jQNgpjWca8HvGIR8zu/Y3WrE8TbjchpaGfWaWQsMOKSBYR/6qshbJyk87VmmQEs6D1Dyth7NAYd6N7pvUlpZzZaOuTJz/5yXMeWD7sGRj7dyx/P93n+0cQEYPPzeDwurZY1cimMFcuIMuspA88IZ5MWHX2gQCjrdFSvY2mRnUjZrEobThYPHmftV5sSr93CpJRxs+skrRpt/yIXUtK0ul1gjPBGWte97rXbVEqZZtZgzcXI3exKB1mtE995GFXXC2up0Lok/ryDne4Q498OUioPmrXRRuKVev4Rn2zmTz/qOzhzHy1zmhxczLkpVB7KSPHUdO92aSItetaa5V2+SmdHY+UbMrWX+fVhiRxzS7xsCLemJ/n/Qb5ZjV4Tc94xjOmEn70dTPKw3UJxMcwxfpwttDW2qOPzsr7/dYRY33SKT48wi5FYLMEtODYgwCk4siuTGa6rbTKkN4ggDDcbwTYI2yPuNgY4Ni4G97whtP1rne92ao0rNeq/LAtNcL1dltIiJXB/tFZIKhVEMTWBkRh5yE2mLYcIH4EXgKGzndVHuM7vrfEhfDu6YKwj/qHZcEIrGX1K4Kzm0+8/UwowVn9VEKV1ovYopv3Y9vG+3UJJEhG6ZQR6SDNHhr/4z/+431Gdxpt/fS93/u9jTzeGd2N+MXSdBRbXI2OIYxiJ3rvM6XTOOIgBAhdkp/2a5X8ISWipLADEK0WtD17WPeUmLTfG81qwdsIZtYywiMyA1JA/WvB3CO7dYC6eTabIQ5lW5+oB946YA1hlLT9GVASQmB8OJMV66Egszhmt3fWeYEIQh0gvfaJyxzGeoa3dW1Rp3e84x29RTjl6RuDp3KDMwnbL9fUS79Rzup/CtTxm66sayVcCTUq9XuSK/wusSCoDurrfv5LHfHO9XGbd66PN0utIoUj9bnjHe/Y4fEmQs5fHbWoBXSvT4qtaHHtcnuVIX8ntJb9UkuOchCQ9Bv98PH6lsJSmcTB1it0JiRPdXZfG4EWy9YSMmVlQ9BGeQsnenaUXQ0QCz7JlDWKrcf20LcwkuRlJXmnP4qw+h0zFG0m7iYyFU+YXXzu1Ts4k/4fy9gP96kfsyJ1dgV5v6qOa0qxMsIYYSuDPte6Mj1QUB9qlvIYObAbIG3DijhJ18jInCQzB77bKbtG+IC8gLTVkX21L8NPPNM1c5B71B4OIz2WjncSM4lp3YhlnWCGMWNZN5gVsClmkJiG8NQhzIgdMOv5Btg15jDWHjwyKqOEJ9NZJZ0jmbHmUYa1jnp5NkKOoO6pf64kcX5mALMB0wwslPRYP+weL+tmQvlqJ1AnbCgw48hvP0O+u3Yyryn9TveX9Wb64qT6V8BJkBHAyHLrW996ESWUiAk7KdE+epE6Uo6V66EeLarh85U0I8oj70mMSIrc00xrNzCy+CW/sYne+a0VXgTR+gUacSO40YqUi55ihCKQRSFnl222q/VeB9P4MxdRJ8pFGu2Acmm3HVdGIuNKd6HM5bqmjmvVU3jiJH91ysxR66Y2PVEPWndlsDDwPP7iHDB57Ndr+oeuitI4/a1/VgHKOQnGTHTCRpmclME+eJE21MjWWvEa2dvMIooiotIoCkMcWMmSDnXtIcJWQHl+ayHicl7JH4JTWupnLFoA+0cZ5z02KwSSdIm36rqVeozpRyRhMaBsfRO20355bJofIrn3ve+9YOdF8Zl6pd/HfPfTfdrIyqH8aTVurFe/kwgkDWRGwKS7xGKzljVh62W4n8LG+poVoh1WR/x8CRwW5VV91nCzN5Imv1NpS/JYdZVv
```

Comment:
90% of this text is common between the different paymentOptions please move it up out of the if